src/types/hexColorCode.ts

```diff
@@ -0,0 +1,12 @@
+import {StrongType, makeStrong} from '../strong-type';
+
+import {STRules} from '../rules';
+
+export type StrongHexCode = StrongType<string>;
+
+export function makeHexCode(initial: string | null | undefined, fallback: string): StrongHexCode {
```

Comment:
make functions should match the file type. I think we missed this one when changing naming throughout the project.

- [ ] `makeHexCode` should be `makeHexColorCode` (thats a mouthful) for consistency.